packages/queries/src/query.ts

```diff
@@ -8,9 +8,9 @@ export class QueryBuilder<
     DB,
     TTable extends Table<DB>,
     TWhere extends Where<DB, TTable>,
-    TCardinality extends 'one' | 'maybe' | 'many',
-    TInclude extends Include<DB>,
     TSelect extends Select<DB, TTable>,
+    TInclude extends Include<DB>,
```

Comment:
As I've mentioned before @jimezesinachi, try to avoid these refactorings. They just make the PR longer to review but don't really add anything.

Reply:
I agree! But this one was really necessary. The difference in the argument order caused some confusion and time when I was working on the `from`. I only changed it because of that.